lib/presentation/components/post_card/bottom_sheets/custom_bottom_sheet.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:gap/gap.dart';
+import 'package:social_mobile/domain/post/post.dart';
+import 'package:social_mobile/i18n/strings.g.dart';
+import 'package:social_mobile/presentation/components/post_card/contents_only_post_card.dart';
+import 'package:social_mobile/utils/extensions/context.dart';
+
+class CustomBottomSheet extends StatelessWidget {
+  const CustomBottomSheet({
+    super.key,
+    required this.post,
+    required this.iconPath,
+    required this.title,
+    required this.buttonText,
+    required this.onPressedButton,
```

Comment:
こちらも何の処理を書くのか直感でわかりやすくするために、変更しましょう！
```suggestion
    required this.onPressedSubmit,
```